qtp_biom/summary.py

```diff
@@ -78,7 +78,7 @@ def _generate_html_summary(biom_fp, metadata, out_dir, is_analysis, tree=None):
             1
             for tip
             in tree.tips()
-            if (tip.name is not None) and (not tip.name.isdigit())])
+            if (tip.name is not None) and tip.name[0] in 'ATGC'])
```

Comment:
Just to confirm, this is fine for all trees? Well, at least the deblur generated and wolr1-2 trees?

Reply:
The existing solution wasn't fine for all trees :) but this is comparable to that solution, which I believe was written with the intention of disregarding Greengenes 13_8 identifiers

Reply:
Can remember but sounds good; trust you.